src/Components/Components/ref/Microsoft.AspNetCore.Components.csproj

```diff
@@ -5,13 +5,12 @@
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <Compile Include="Microsoft.AspNetCore.Components.netstandard2.0.cs" />
-    <Reference Include="Microsoft.JSInterop"  />
     <Reference Include="Microsoft.Extensions.Logging.Abstractions"  />
     <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions"  />
+    <Reference Include="System.Buffers"  />
```

Comment:
hmm. Why does this show up here now? Were we getting this from JS interop before?